app/src/views/private/components/export-sidebar-detail/export-sidebar-detail.vue

```diff
@@ -65,13 +66,21 @@ export default defineComponent({
 			required: true,
 		},
 	},
+
 	setup(props) {
 		const { t } = useI18n();
 
 		const format = ref('csv');
 		const useFilters = ref(true);
+		const collections = useCollectionsStore();
+		let collectionName;
+		collections.visibleCollections.forEach((collection) => {
+			if (collection.collection === props.collection) {
+				collectionName = collection.name;
+			}
+		});
```

Comment:
Use `collectionsStore.getCollection()` instead https://github.com/directus/directus/blob/af4dbad27d6f96a592959325dadc2f06946fdbd6/app/src/stores/collections.ts#L110-L112

(ps, instead of forEach with an if-statement, it's easier to use a `.find()` 🙂